src/components/posts/buttons/DeleteButton.js

```diff
@@ -1,16 +1,46 @@
-function DeleteButton(props) {
+import { useState } from "react/cjs/react.development";
+import Backdrop from "../../UI/Backdrop";
+import Post from "../../UI/Post";
+import classes from "./EditModal.module.css";
+
+function DeleteButton({ id, setPostDelete }) {
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   function deletePost() {
     fetch(
-      `https://friendface-react-e85cf-default-rtdb.firebaseio.com/posts/${props.id}.json`,
+      `https://friendface-react-e85cf-default-rtdb.firebaseio.com/posts/${id}.json`,
       {
         method: "DELETE",
       }
     ).then(() => {
-      props.setRefreshPost(true);
+      setShowDeleteAlert(false);
+      setPostDelete(false);
     });
   }
+  function deleteAlert() {
+    if (showDeleteAlert) {
```

Comment:
This `if` statement can be removed and reduced to a more concise function

Reply:
showDeleteAlert = !showDeleteAlert;

Reply:
right?